test/units/TEST-75-RESOLVED.sh

```diff
@@ -66,11 +68,14 @@ restart_resolved() {
 }
 
 # Test for resolvectl, resolvconf
+manual_testcase_01_resolvectl() {
 systemctl unmask systemd-resolved.service
```

Comment:
This is a "global" setup, not a part of the test case.

Reply:
I did not want to overcomplicate it, but you are right. Fixed, thanks!